native_client/dotnet/DeepSpeechClient/DeepSpeech.cs

```diff
@@ -209,18 +209,6 @@ public unsafe void FreeStream()
             NativeImp.DS_FreeStream(ref _streamingStatePP);
         }
 
-        /// <summary>
-        /// Free a DeepSpeech allocated string
-        /// </summary>
-        public unsafe void FreeString(IntPtr intPtr)
-        {
-            NativeImp.DS_FreeString(intPtr);
```

Comment:
This is only used with private members while marshaling, we should not expose them.